SensorController/Core/Src/main.c

```diff
@@ -118,10 +126,12 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_DMA_Init();
-
+  MX_ETH_Init();
```

Comment:
This shouldn't be called here, probably CubeMX doing this

Reply:
yeah, I didn't remove it because I wasn't sure when we needed it